src/rgw/rgw_op.cc

```diff
@@ -1444,13 +1444,23 @@ void RGWListBuckets::execute()
 
   uint64_t max_buckets = s->cct->_conf->rgw_list_buckets_max_chunk;
 
+  rgw_user *accounts_user;
+  rgw_user buckets_user;
+
```

Comment:
Please move the following
```
  op_ret = get_params();
  if (op_ret < 0) {
    goto send_end;
  }
```
from below to here, the `doing_swift_cross_tenant` variable is initialized by the `get_params()` function.